Source/Utils/ScreenCover.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ScreenCover.swift
+//  walmart
+//
+//  Created by David Sica on 5/9/16.
+//  Copyright © 2016 Walmart. All rights reserved.
+//
+
+import Foundation
+
+// Declaring conformance to this Protocol will whitelist the UIViewController's
+//  view.
+//
+//  Optionally, implement the shouldWhitelist method to determine whether to
+//  wholesale whitelist at runtime.  Or, declare to NOT whitelist but manually 
+//  hide sensitive information on the view.
+@objc protocol ScreenCoverProtocol {
+    optional func shouldWhitelist() -> Bool
```

Comment:
Also, I implemented it thinking I would use it for RxAPI stuff that I took out but then realized we should just wholesale blacklist RxAPI for now given the requirements of the CB-5551 ticket.